bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v2/form/FormContainerImpl.java

```diff
@@ -190,6 +190,11 @@ public String getId() {
         }
     }
 
+    @Override
+    public String getName() {
```

Comment:
name is not supported in the json schema as of now.
we should either have a custom prop or have first class support for name.